src/main/java/uk/gov/pay/api/model/CreateCardPaymentRequest.java

```diff
@@ -165,6 +171,12 @@ public String getReturnUrl() {
         return Optional.ofNullable(internal);
     }
 
+    @JsonProperty("set_up_agreement")
+    @Schema(description = "agreement ID", required = false, example = "abcefghjklmnopq1234567890", hidden = true)
```

Comment:
Still only 25 characters…

Reply:
I think you are right, we forgot -n

echo -n abcefghjklmnopq1234567890 | wc -c
      25